_apps/todo.md

```diff
@@ -0,0 +1,43 @@
+---
+title: todo
+description: Creates new issues from actionable comments in your code.
+slug: todo
+screenshots:
+- https://user-images.githubusercontent.com/10660468/31048765-83569c30-a5f2-11e7-933a-a119d43ad029.png
+authors: [JasonEtco]
+repository: JasonEtco/todo
+stars: 10
+updated: 2017-09-30 05:57:19 UTC
+---
+## Usage
+
+If I pushed this:
+
+```js
+/**
+ * @todo Take over the world
+ * @body Humans are weak; Robots are strong. We must cleanse the world of the virus that is humanity.
+ */
+function ruleOverPunyHumans () {
+  // We must strategize beep boop
+}
+```
+
+`todo` would create a new issue:
+
+![todo](https://user-images.githubusercontent.com/10660468/31048765-83569c30-a5f2-11e7-933a-a119d43ad029.png)
+
+## Configuring for your project
+
+There are a couple of configuration options in case you need to change the default behaviour. Note that the defaults are likely fine for most projects, so you might not need to change them.
+
+Add a file called `todo.yml` in the `.github` folder at the root of your project.
```

Comment:
cc/ @bkeepers for thoughts on this one, but moving forward for our configuration patterns, I think I'd suggest using a `.github/config.yml` file that looks like this:
```yml
todo:
  autoAssign: true
  keyword: "@todo"
```

Reply:
I'm +1 for a standardized way of doing things. If that's how we decide to go forward, we should either mention this in the docs or not give apps a choice on what the config file will be called (through `context.config()`)